docs/releases/3.2.txt

```diff
@@ -95,6 +95,9 @@ Minor features
   constrained environments. If this is the case, the existing hasher can be
   subclassed to override the defaults.
 
+* Password reset tokens now incorporate the user's email address, if available.
+  Altering the user's email will now invalidate existing reset tokens.
```

Comment:
Maybe:
```suggestion
* The password reset mechanism now invalidates a token when the user email is
  changed.
```

That's not a new feature, we should move this to the `Backwards incompatible changes in 3.2 -> Miscellaneous`.